app/services/malware_scanner.rb

```diff
@@ -36,12 +35,16 @@ def virus_found?
     @virus_found ||= !scan_result.status.success?
   end
 
+  def scanner_down?
+    scan_result.status.exitstatus != 2
+  end
+
   def scan_result
     @scan_result ||= begin
       stdout, stderr, status = Open3.capture3(*(command + [file_path.to_s]))
       result = stdout.strip
 
-      raise ClamdscanError, (stderr.presence || stdout) if status.exitstatus == 2
+      Raven.capture_message("ClamdscanError, #{stderr.presence || stdout}") if status.exitstatus == 2
```

Comment:
Could you reuse the method from line 38 here?

`Raven.capture_message("ClamdscanError, #{stderr.presence || stdout}") unless scanner_working?`

It would mean we're not explictly checking `exitstatus` in multiple places.